xml/depl_maintenance.xml

```diff
@@ -312,6 +312,175 @@
   </orderedlist>
  </sect1>
 
+ <sect1 xml:id="sec.depl.maintenance.recover.compute.node.failure">
+  <title>Recovering from &compnode; Failure</title>
+  <para>
+   The following procedure assumes that there is at least one &compnode;
+   already running. Otherwise, see <xref
+   linkend="sec.depl.maintenance.bootstrap.compute.plane" />.
+  </para>
+  <procedure>
+   <step>
+    <para>
+     If the &compnode; failed, it should have been fenced. Verify that this is
+     the case. Otherwise, check <filename>/var/log/pacemaker.log</filename> on
+     the Designated Co-ordinator to determine why the &compnode; was not fenced. The
+     most likely reason is a problem with STONITH devices.
+    </para>
+   </step>
+   <step>
+    <para>
+     Determine the cause of the &compnode;'s failure.
+    </para>
+   </step>
+   <step>
+    <para>
+     Boot the &compnode; again.
+    </para>
+   </step>
+   <step>
+    <para>
+     Check whether the <systemitem>crowbar_join</systemitem> script ran
+     successfully on the &compnode;. If this is not the case, check the log
+     files to find out the reason.
+    </para>
+   </step>
+   <step>
+    <para>
+     If the <systemitem>chef-client</systemitem> agent triggered by
+     <systemitem>crowbar_join</systemitem> succeeded, confirm that the
+     <systemitem>pacemaker_remote</systemitem> service is up and
+     running.
+    </para>
+   </step>
+   <step>
+    <para>
+     Check whether the remote node is registered and considered healthy by the
+     core cluster. If this is not the case check
+     <filename>/var/log/pacemaker.log</filename> on the Designated Co-ordinator
+     to determine the cause. There should be a remote primitive running on the
+     core cluster (active/passive). This primitive is responsible for
+     establishing a TCP connection to the
+     <systemitem>pacemaker_remote</systemitem> service on the &compnode;. Use
+     standard network debugging methods to make sure that nothing is preventing
+     this particular TCP connection from being established (for example, problems with
+     NICs, switches, firewalls etc.).
+    </para>
+   </step>
+   <step>
+    <para>
+     If Pacemaker can communicate with the remote node, it starts the
```

Comment:
I think "it should try to start" is better wording in this trouble-shooting context, since we are dealing with things which have gone wrong, so we can't assume that any single step always works.